provider/server/machine_annotations_update_test.go

```diff
@@ -36,12 +36,9 @@ var _ = Describe("UpdateMachineAnnotations", func() {
 		Expect(createResp).NotTo(BeNil())
 
 		DeferCleanup(func(ctx SpecContext) {
+			_, err := machineClient.DeleteMachine(ctx, &iri.DeleteMachineRequest{MachineId: createResp.Machine.Metadata.Id})
+			Expect(err).To(Succeed())
```

Comment:
I believe directly asserting the function call's result for success is more concise and easier to read.
eg: 
```suggestion
Expect(machineClient.DeleteMachine(ctx, &iri.DeleteMachineRequest{MachineId: createResp.Machine.Metadata.Id})).To(Succeed())

```

Reply:
`DeleteMachine` returns 2 values, so its not possible